appinventor/appengine/war/Ya.css

```diff
@@ -990,6 +990,65 @@ select.ode-PropertyEditor[disabled] {
   border: 2px solid black;
 }
 
+
+.ode-SimpleMockFormPhoneLandscape {
+    background: url(images/phoneLandscape.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 100% 100%;
+    /*padding: 5px 54px 5px 54px;*/
```

Comment:
Remove commented out styles here and elsewhere in this patch.

Reply:
Done